pkgs/applications/networking/cluster/sonobuoy/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name    = "sonobuoy-${version}";
```

Comment:
whats up with this indentation? is this from a tool?

Reply:
nope I'm just weird :smile: